src/js/components/campaignpage/AdvertsCatalogue.jsx

```diff
@@ -42,36 +42,18 @@ const viewCount = (viewcount4campaign, ad) => {
 	return null;
 };
 
-/**
- * @param {!Advert} ad 
- * @returns {!string} Can be "unknown" to fill in for no-campaign odd data items
- */
-const campaignNameForAd = ad => {
-	if (!ad.campaign) return "unknown";
-	// HACK FOR TOMS 2019 The normal code returns 5 campaigns where there are 3 synthetic campaign groups
-	// Dedupe on "only the first josh/sara/ella campaign" instead
-	if (ad.vertiser === 'bPe6TXq8' && ad.campaign && ad.campaign.match(tomsCampaigns)) {
-		let cname = ad.campaign.match(tomsCampaigns)[0];
-		return cname;
-	}
-	return ad.campaign;
-};
-
 /**
  * List of adverts with some info about them (like views, dates)
  * @param {*} param0 
  */
-const AdvertsCatalogue = ({campaign, ads, viewcount4campaign, donationTotal, nvertiserName, totalViewCount, showNonServed }) => {
+const AdvertsCatalogue = ({campaign, ads, viewcount4campaign, donationTotal, nvertiserName, totalViewCount}) => {
 	// filter out any hidden ads
 	// NB: done here as the hiding is a shallow cosmetic -- we still want the view and £ donation data included (or if not, there are other controls)
```

Comment:
Hello @dodo721 If you remove some code -- also remove the comments that go with it (or modify them to reflect the new status)